server/gitrest/package.json

```diff
@@ -40,7 +40,7 @@
     "nodegit": "^0.26.1",
     "split": "^1.0.0",
     "uuid": "^3.3.2",
-    "winston": "^2.4.4"
```

Comment:
In this package, only `winston.info` and `winston.error` calls are used. Major version update does not affect them.